### PR TITLE
Lovelace Entity Card Editor to Ha Form - Adds Theme Selector and HaFormColumn

### PR DESCRIPTION
**Pull request by zsarnett**
## Proposed change

Add Theme Selector
Add HaFormColumn
Convert Entity Card Editor to HaForm

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [x] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/components/ha-form/ha-form-column.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+              .data=${this.data}
+              .schema=${form}
+              .disabled=${this.disabled}
+              .computeLabel=${this.computeLabel}
+              .computeHelper=${this.computeHelper}
+            ></ha-form>
+          `
+      )}
+    `;
+  }
```
Comment (zsarnett) [inline src/components/ha-form/ha-form-column.ts:45]: The issue with this implementation is the Tab order

**Review by zsarnett — COMMENTED** [review]
LGTM

File: src/panels/lovelace/editor/config-elements/hui-entity-card-editor.ts [hunk 2]
```diff
@@ -100,7 +100,9 @@ export class HuiEntityCardEditor
   }
 
   private _valueChanged(ev: CustomEvent): void {
-    fireEvent(this, "config-changed", { config: ev.detail.value });
+    const config = ev.detail.value;
+    Object.keys(config).forEach((k) => config[k] === "" && delete config[k]);
+    fireEvent(this, "config-changed", { config });
```
Comment (zsarnett) [inline src/panels/lovelace/editor/config-elements/hui-entity-card-editor.ts:105]: Isn't this done by HA Form Automatically?

Comment (bramkragten) [reply src/panels/lovelace/editor/config-elements/hui-entity-card-editor.ts:105]: No, it doesn't remove values

**Review by bramkragten — APPROVED** [review]

**Pull request merged**